phoenix_liveview/phoenix_liveview.exs

```diff
@@ -0,0 +1,134 @@
+Application.put_env(:phoenix, :json_library, Jason)
+Application.put_env(:esbuild, :version, "0.12.18")
+
+Mix.install(
+  deps = [
+    {:phoenix, "~> 1.6.0-rc.0", override: true},
+    :jason,
+    :plug_cowboy,
+    {:phoenix_live_view, "~> 0.16.0"},
+    {:esbuild, "~> 0.2"}
+  ]
+)
+
+Code.require_file("mix_install_deps_backport.exs")
+
+Application.put_env(:esbuild, :default,
+  args: ~w(app.js --bundle --target=es2016 --outdir=../priv/static/assets),
+  cd: Path.expand("assets", __DIR__),
+  env: %{"NODE_PATH" => InstallFolderTemporaryBackport.determine_build_folder(deps) <> "/deps"}
```

Comment:
As noted before, this will be much improved if `Mix.install/2` exposes the computed `deps` to the script.